src/main/java/xyz/nucleoid/dungeons/dungeons/assets/ShittyAssModelGenerator.java

```diff
@@ -0,0 +1,75 @@
+package xyz.nucleoid.dungeons.dungeons.assets;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonObject;
+import net.fabricmc.loader.api.FabricLoader;
+import net.minecraft.item.Item;
+import net.minecraft.util.registry.Registry;
+import xyz.nucleoid.dungeons.dungeons.game.loot.DgModelRegistry;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.LinkedList;
+import java.util.Map;
+
+public class ShittyAssModelGenerator {
```

Comment:
While I really love this name I also think that it should be changed :P

Reply:
Maybe after it's not shitty lol